tests/signatures/ed25519_key_and_ed25519_detects_bad_signature.sig.bin

```diff
@@ -1,2 +1 @@
-͕(/���߽N���*�3}�]Ew� ����D�XI׹ъ�$���ٚ�2t� 
-r-�6ث
\ No newline at end of file
+3��J�B���3%5�����Z~|f[�"���pVƫP���+d�Jod�߫'���BH6�P�&	
```

Comment:
I can't get my new tests to pass without adding the `--force` argument, which causes everything to re-generate. I get `InvalidCrlSignatureForPublicKey` otherwise.